src/main/resources/i18n/messages_de.properties

```diff
@@ -498,7 +498,7 @@ map.maxPlayers=Max. Spieler
 map.width=Breite
 map.height=Höhe
 replayServer.unreachable=Replay-Server ist nicht erreichbar, Spiel kann nicht aufgezeichnet werden
-mapNotFound=Die Karte ''{0}'' konnte weder auf dem Server noch auf diesem Computer gefunden werden.
+mapNotFound=Die Karte ''{0}'' für lokales Replay konnte weder auf dem Server noch auf diesem Computer gefunden werden. Dieses Replay als korrupt markieren und in seperaten Ordner verschieben?
```

Comment:
All translations need to be added to poeditor as well, unfortunately, otherwise they will get overridden

Reply:
On the website?

Reply:
ok then lets have them overwritten

Reply:
And I reenter them on the website as soon as the PR gets merged no?